README.md

```diff
@@ -1,12 +1,110 @@
-# Toolbox Gateway plugin sample
+# Coder Toolbox plugin
 
-To load plugin into the provided Toolbox App, run `./gradlew build copyPlugin`
+[!["Join us onDiscord"](https://img.shields.io/badge/join-us%20on%20Discord-gray.svg?longCache=true&logo=discord&colorB=purple)](https://discord.gg/coder)
+[![Twitter Follow](https://img.shields.io/twitter/follow/CoderHQ?label=%40CoderHQ&style=social)](https://twitter.com/coderhq)
+[![Coder Toolbox Plugin Build](https://github.com/coder/coder-jetbrains-toolbox/actions/workflows/build.yml/badge.svg)](https://github.com/coder/coder-jetbrains-toolbox/actions/workflows/build.yml)
 
-or put files in the following directory:
+Connects your JetBrains IDE to Coder workspaces
 
-* Windows: `%LocalAppData%/JetBrains/Toolbox/cache/plugins/plugin-id`
-* macOS: `~/Library/Caches/JetBrains/Toolbox/plugins/plugin-id`
-* Linux: `~/.local/share/JetBrains/Toolbox/plugins/plugin-id`
+## Getting Started
 
-Put all required .jar files (do not include any dependencies already included with the Toolbox App to avoid possible resolution conflicts),
-`extensions.json` and `icon.svg` in this directory.
+To install this plugin using JetBrains Toolbox, follow the steps below.
+
+1. Install [JetBrains Toolbox](https://www.jetbrains.com/toolbox-app/). Make sure it's the `2.6.0.40284` release or
+   above.
+2. Launch the Toolbox app and sign in with your JetBrains account (if needed).
+
+### Install Coder plugin via URI
+
+You can quickly install the plugin using this JetBrains hyperlink.
+
+👉 [Install plugin](jetbrains://gateway/com.coder.toolbox)
+
+This will open JetBrains Toolbox and prompt you to install the Coder Toolbox plugin automatically.
+
+Alternatively, you can paste `jetbrains://gateway/com.coder.toolbox` into a browser.
+
+### Manual install
+
+There are two ways Coder Toolbox plugin can be installed. The first option is to manually download the plugin
+artifact from [JetBrains Marketplace](https://plugins.jetbrains.com/plugin/26968-coder/versions)
+or from [Coder's Github Release page](https://github.com/coder/coder-jetbrains-toolbox/releases).
+
+The next step is to copy the zip content to one of the following locations, depending on your OS:
+
+* Windows: `%LocalAppData%/JetBrains/Toolbox/cache/plugins/com.coder.toolbox`
```

Comment:
```suggestion
* Windows: `%LocalAppData%/JetBrains/Toolbox/plugins/com.coder.toolbox`
```
I think this was a typo and IIRC the last time I tested on Windows the correct path was this.